cmd/ebrelayer/txs/relayToEthereum.go

```diff
@@ -185,4 +107,97 @@ func InitRelayConfig(
 
 	}
 	return client, transactOptsAuth, target, nil
-}
\ No newline at end of file
+}
+
+// RelayProphecyCompletedToEthereum send the prophecy aggregation to CosmosBridge contract on the Ethereum network
+func RelayProphecyCompletedToEthereum(
+	prophecyInfo types.ProphecyInfo,
+	sugaredLogger *zap.SugaredLogger,
+	client *ethclient.Client,
+	auth *bind.TransactOpts,
+	cosmosBridgeInstance *cosmosbridge.CosmosBridge,
+) error {
+
+	// Send transaction
+	sugaredLogger.Infow(
+		"Sending SubmitProphecyClaimAggregatedSigs to CosmosBridge.",
+		"CosmosSender", prophecyInfo.CosmosSender,
+		"CosmosSenderSequence", prophecyInfo.CosmosSenderSequence,
+		"ProphecyId", prophecyInfo.ProphecyID,
+	)
+
+	claimData := cosmosbridge.CosmosBridgeClaimData{
+		CosmosSender:         []byte(prophecyInfo.CosmosSender),
+		CosmosSenderSequence: big.NewInt(int64(prophecyInfo.CosmosSenderSequence)),
+		EthereumReceiver:     common.HexToAddress(prophecyInfo.EthereumReceiver),
+		TokenAddress:         common.HexToAddress(prophecyInfo.TokenSymbol),
+		Amount:               &prophecyInfo.TokenAmount,
+		DoublePeg:            prophecyInfo.DoublePeg,
+		Nonce:                big.NewInt(int64(prophecyInfo.GlobalNonce)),
```

Comment:
You'll also need to add the network descriptor here.

@daniportugalgit please take a look over this claim data and check if anything is missing.

Reply:
Apart from the `networkDescriptor`, the new claim data structure has 3 other new fields: `tokenName`, `tokenSymbol` and `tokenDecimals`. Here's the typed struct we will have on our smart contracts once my PR gets merged into peggy2:
```
struct ClaimData {
  bytes cosmosSender;
  uint256 cosmosSenderSequence;
  address payable ethereumReceiver;
  address tokenAddress;
  uint256 amount;
  bool doublePeg;
  uint128 nonce;
  uint256 networkDescriptor;
  string tokenName;
  string tokenSymbol;
  uint8 tokenDecimals;
}
```

@ElliotFriedman By the way, I _think_ we might save some gas by reordering those properties inside the struct (because of tight variable packing). It's just optimization, though, and we could leave it to some other time.

Reply:
sure, after the new smart contract merged. will generate new go source file and add these items.